tests/smoketests.rs

```diff
@@ -122,3 +122,29 @@ fn malformed_file_from_env() {
 
     env::remove_var("SSL_CERT_FILE");
 }
+
+#[test]
+#[serial]
+fn from_env_missing_file() {
+    env::set_var("SSL_CERT_FILE", Path::new("no/such/file"));
```

Comment:
Separate from the idea of a helper fn, I think L185 could use `no/such/directory` directly right? That would save the `Path` import being added on L3.

Reply:
I still see `Path` being imported and used on L129 of d4ffe71475cb748703d14ed00d1e646d28ced696 Did that change get dropped?